xc7/primitives/slicem/srl/srl32.pb_type.xml

```diff
@@ -0,0 +1,32 @@
+<pb_type name="SRL32" num_pb="1" blif_model=".subckt SRL32" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Wrong subckt name?  Should be SRLC32E? Typical VPR specific naming would be SRLC32E_VPR?

Reply:
Done.